source/docs/software/wpilib-tools/robot-characterization/analyzing-feedback.rst

```diff
@@ -47,6 +34,8 @@ To specify the correct settings for your PID controller, use the following optio
 - **Has Slave:** Whether there is a motor controller slaved to the controller running the control loop, if the control loop is being run on a peripheral device. This changes the effective loop period.
 - **Slave Update Period:** The rate at which the slave (if present) is updated. By default, this is 100Hz (every 0.01s) for Talons and Sparks, but can be changed.
 
+.. note:: If you select a smart motor controller as the preset (e.g. TalonSRX, Spark MAX, etc.) the Convert Gains checkbox will be automateically checked. This means the tool will convert your gains so that they can be used through the smart motor controller's PID methods. Therefore, if you would like to use WPILib's PID Loops, you must uncheck that box.
```

Comment:
```suggestion
.. note:: If you select a smart motor controller as the preset (e.g. TalonSRX, Spark MAX, etc.) the :guilabel:`Convert Gains` checkbox will be automatically checked. This means the tool will convert your gains so that they can be used through the smart motor controller's PID methods. Therefore, if you would like to use WPILib's PID Loops, you must uncheck that box.
```